src/components/OrgPicker/OrgPicker.tsx

```diff
@@ -90,12 +110,12 @@ export function OrgPicker({ single }: { single: Boolean }) {
         aria-expanded={isOpen}
         ref={orgsSelectorRef}
       >
-        <header className="orgs-selector__header display-flex desktop:padding-y-1 flex-align-center">
-          <strong className="orgs-selector__current text-bold text-base-darker text-ellipsis margin-right-1 padding-right-1 border-right border-base-light">
-            sandbox-gsa-much-longer-name-goes-here-and-is-very-very-long
+        <header className="orgs-selector__header display-flex desktop:padding-y-1 flex-align-center flex-justify">
+          <strong className="orgs-selector__current text-bold text-base-darker text-ellipsis margin-right-1 padding-right-1">
+            {currentOrg?.name || 'loading'}
           </strong>
           <button
-            className="usa-button usa-button--unstyled width-5 flex-justify-center"
+            className="usa-button usa-button--unstyled width-5 flex-justify-center border-left border-base-light"
```

Comment:
Not sure where to log this screenshot in this PR, but it looks like there’s a gap between the picker’s button and the container it sits in:

<img width="339" alt="Screenshot 2024-09-26 at 2 02 22 PM" src="https://github.com/user-attachments/assets/d9b37af1-e5ae-4336-9d0c-e4168f8e7498">

(It should look a little more square/even, like the below.)

<img width="349" alt="image" src="https://github.com/user-attachments/assets/24b8116c-d16c-409d-950d-f4cc4b258f84">

Reply:
Thanks! Some right padding got in there, so I removed it:

<img width="536" alt="Screenshot 2024-09-26 at 2 16 10 PM" src="https://github.com/user-attachments/assets/81ff6c4d-f0ca-4010-9ad0-37a64d6de071">

Reply:
The button looks fantastic now! Unfortunately, that seems to have clobbered some padding inside the menu itself:

<img width="364" alt="Screenshot 2024-09-26 at 3 47 57 PM" src="https://github.com/user-attachments/assets/1a108aae-2e2d-4e77-afc1-b444f4657296">

Reply:
Aye, I spoke too soon.... question: do we need the entire right side to be clickable? (as in, the area in pink in this screenshot)

<img width="362" alt="Screenshot 2024-09-26 at 3 13 07 PM" src="https://github.com/user-attachments/assets/a76cfef7-3e6b-441b-8924-e681c498b1dc">

Reply:
_Ideally_, that’d be great, as it’d give us a nice generous tap area for touch. But let me know what you think is doable.

Reply:
I would need to re-think the structure of the padded areas for the whole picker, which I'm happy to do but it'll take some more time. 

What I can do now as a stop-gap is put back the right padding that I took off, and then expand the left padding of the button, so that the icon looks more centered between the left border and the right end. That should give a wider clickable area even with the right outer padding put back. How does this look?

<img width="494" alt="Screenshot 2024-09-26 at 3 30 11 PM" src="https://github.com/user-attachments/assets/1af0532e-825b-4ba3-a800-56bb88bb77c0">

<img width="540" alt="Screenshot 2024-09-26 at 3 29 58 PM" src="https://github.com/user-attachments/assets/265e1ab4-0d6f-43d6-953f-c290e158621a">

Reply:
Yep, that feels like the right approach — thanks, Eleni! Let’s go with that, and we can tackle the structural question in a separate ticket.

Thanks for talking this through with me!

Reply:
Thank _you_!